pagerdutyplugin/data_source_pagerduty_standards_resource_scores.go

```diff
@@ -0,0 +1,130 @@
+package pagerduty
+
+import (
+	"context"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type dataSourceStandardsResourceScores struct {
+	client *pagerduty.Client
+}
+
+var _ datasource.DataSource = (*dataSourceStandardsResourceScores)(nil)
+
+func (d *dataSourceStandardsResourceScores) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = "pagerduty_standards_resource_scores"
+}
+
+func (d *dataSourceStandardsResourceScores) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id":            schema.StringAttribute{Required: true},
+			"resource_type": schema.StringAttribute{Required: true},
```

Comment:
Input options validation is needed here. Currently the only supported input is `technical_services`.